docs/reference/commandline/swarm_join.md

```diff
@@ -22,6 +22,7 @@ Join a swarm as a node and/or manager
 
 Options:
       --advertise-addr string   Advertised address (format: <ip|interface>[:port])
+      --datapath-addr string    Data path address (format: <ip|interface>)
```

Comment:
The usage info here should match what's printed by `swarm join --help`. I believe the flags are sorted alphabetically in `swarm join --help`.